migrate/20240125_add_minio_crts.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  up do
+    alter_table(:minio_cluster) do
+      add_column :root_cert_1, :text, collate: '"C"'
+      add_column :root_cert_key_1, :text, collate: '"C"'
+      add_column :root_cert_2, :text, collate: '"C"'
+      add_column :root_cert_key_2, :text, collate: '"C"'
+      add_column :certificate_last_checked_at, :timestamptz, null: false, default: Sequel.lit("now()")
+    end
+
+    alter_table(:minio_server) do
+      add_column :cert, :text, collate: '"C"'
+      add_column :cert_key, :text, collate: '"C"'
+    end
+  end
+
+  down do
```

Comment:
I think you don't need separate down block. All the operations here can be reversed by Sequel automatically if you use change block above.